tests/e2e/2-api-endpoints/create-and-delete.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { APIResponse, expect, test } from "@playwright/test";
+import { execSync } from "child_process";
+import { resolve } from "path";
+import { sleep } from "../globals";
+
+const devAppPath = resolve("../webstone-dev-app");
+
+test.describe("web/api/create & web/api/delete", () => {
+  test("creates and deletes CRUD API endpoints for /api/users", async ({
+    request,
+  }) => {
+    execSync("pnpm ws web api create /api/users", {
+      cwd: devAppPath,
+    });
+
+    let response: APIResponse;
+    response = await request.get("/api/users");
+    expect(response.ok()).toBeTruthy();
+    expect(response.status()).toEqual(200);
+    expect(await response.text()).toEqual("GET /api/users => Ok.");
+
+    response = await request.post("/api/users");
+    expect(response.ok()).toBeTruthy();
+    expect(response.status()).toEqual(200);
+    expect(await response.text()).toEqual("POST /api/users => Ok.");
+
+    response = await request.delete("/api/users/123");
+    expect(response.ok()).toBeTruthy();
+    expect(response.status()).toEqual(200);
+    expect(await response.text()).toEqual("DELETE /api/users/123 => Ok.");
+
+    response = await request.patch("/api/users/123");
+    expect(response.ok()).toBeTruthy();
+    expect(response.status()).toEqual(200);
+    expect(await response.text()).toEqual("PATCH /api/users/123 => Ok.");
+
+    response = await request.put("/api/users/123");
+    expect(response.ok()).toBeTruthy();
+    expect(response.status()).toEqual(200);
+    expect(await response.text()).toEqual("PUT /api/users/123 => Ok.");
+
+    execSync("pnpm ws web api delete /api/users", {
+      cwd: devAppPath,
+    });
+
+    await sleep(300);
```

Comment:
I have to call the sleep function here, as the dev-server is restarting after the call before. I wonder if there is a cleaner way to do this.

Reply:
That's fine, I documented the Why? in the code directly so that future contributors understand why this exists.